spec/unit/type/gluster_peer_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
Are you sure about the `spec/unit/type` path? Other things are just under `spec` like `spec/classes`, `spec/defines`, etc... I guess this is Ruby code so it's a bit different but just checking...